### ISSUE TITLE
I don't really have an issue but I would just like to say...

### ISSUE THREAD

**Issue by kaleidoscope-dev** [leandromoreira/ffmpeg-libav-tutorial#141]
You're on point in everything.  I wish I had something like this when I was trying to figure it all out.    

Did you ever think about  continuing the tutorial by integrating Dranger's simple video player examples?

**Comment by leandromoreira** [comment]
thank you @kaleidoscope-dev kind of you.

> Did you ever think about continuing the tutorial by integrating Dranger's simple video player examples?

yeah, I want to but life/work is in the way 🤣 🤣 🤣

**Comment by clort81** [comment]
Thank you. Learned what i needed and saved time.  Thanks.

**Comment by JackLau1222** [comment]
> thank you [@kaleidoscope-dev](https://github.com/kaleidoscope-dev) kind of you.
> 
> > Did you ever think about continuing the tutorial by integrating Dranger's simple video player examples?

Good idea, and I already do such work in https://github.com/JackLau1222/Simple-AV-Synchronization-Player

> 
> yeah, I want to but life/work is in the way 🤣 🤣 🤣

Hi @leandromoreira, I'd like to continue to improve this tutorial.

This is my personal tutorial repo: https://github.com/TSGU-OSC/Learn_FFmpeg_the_Hard_Way
And I already contributed two commits into FFmpeg codebase
So I really like this field and I am passionate about open source.
But I need a precise goal that how to improve this tutorial. For example, we can add more tutorial code about filter like this https://github.com/TSGU-OSC/Learn_FFmpeg_the_Hard_Way/blob/master/code/demo/filter_link_video.c or something else about FFmpeg api

**Comment by leandromoreira** [comment]
@JackLau1222 thanks for the offer :) 

maybe the natural flow would be:

1. add filtering (using newer codec like av1)
2. you can go here and explore the ffmpeg api briefly but still useful
   2.a. you may compare it to gpac and other similar full-fledge media programs
3. packaging - given an input (network, file) you generate http streaming protocols (hls, dash, if possible low latency)
wdyt?
4. maybe finish with a simple player/server exposing the main features
